updated Home-Schedule-Sponsor

vtushar06 opened the pull request:
<!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
<!-- Ideally, a GitHub Issue with the label `Status: Consensus` will have been concluded, and linked to in this PR. -->
<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**
Feature enhancement and content update
<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**
<!-- Pick one of the below options.  Please remove those which don't apply. -->
-  Closes #65  <!-- Replace ___ with the issue number this PR resolves -->

**Screenshots/videos:**


<img width="1512" height="982" alt="Screenshot 2025-09-01 at 7 48 53 PM" src="https://github.com/user-attachments/assets/6fc4153a-cfc9-4279-864b-05f54b5e73bc" />

<img width="1512" height="982" alt="Screenshot 2025-09-01 at 7 49 06 PM" src="https://github.com/user-attachments/assets/8df526d9-5dea-4c75-94e7-aa604622d051" />

<img width="1512" height="982" alt="Screenshot 2025-09-01 at 7 49 18 PM" src="https://github.com/user-attachments/assets/8bf3c681-2d05-4ffa-b1eb-88d44f7beb06" />



<!--Add screenshots or videos wherever possible.-->

**If relevant, did you update the documentation?**

<!--Add link to it-->

**Summary**

This PR updates the JSON Schema Conference website to reflect the 2025 Paris conference. The changes include:


1. **Home Page Updates:**

- Updated page title to reflect “JSON Schema Conference 2025
- Removed redundant Home tab and made logo clickable to route to home
- Implemented “Apply to Speak” CTA button
- Added “About the Event” section
- Included “Become a Sponsor” CTA
- Added “Meet Speakers” section
- Implemented “Tickets” CTA button
- 
2. **Schedule Page:**
- Added “Coming Soon” placeholder content since the schedule is not yet finalized

3. **Sponsors Page:**
- Added “Coming Soon” section for 2025 sponsors

<!-- Explain the motivation for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

**Does this PR introduce a breaking change?**
No

vtushar06 committed:
updated Home-Schedule-Sponsor

jagpreetrahi commented:
Hey @vtushar06, I hope you’re doing well!
I just checked the provided outcomes from your side (via screenshots) and haven’t explored the code yet, but I have a few suggestions:

1. Navbar Layout
In the first screenshot, should we add more spacing between the logo and the list items in the navbar?
Also, regarding the conference data (days, sessions, etc.) on the right side: currently, the spacing feels a bit too large. Would it look better if we displayed them horizontally instead of stacking with so much space?

2. Button Sizes
In the second screenshot, the buttons seem larger than necessary. Do we really need them at that size?
I think we could minimize the button size so that it fits just around the content for a cleaner look.

jagpreetrahi commented:
hey @Honyii @vtushar06  , Can we show the participant number dynamically instead of statically? It would enhance the page view. What do you think about it??

Utkarsh-123github commented:
> Hey @vtushar06, I hope you’re doing well! I just checked the provided outcomes from your side (via screenshots) and haven’t explored the code yet, but I have a few suggestions:
> 
> 1. Navbar Layout
>    In the first screenshot, should we add more spacing between the logo and the list items in the navbar?
>    Also, regarding the conference data (days, sessions, etc.) on the right side: currently, the spacing feels a bit too large. Would it look better if we displayed them horizontally instead of stacking with so much space?
> 2. Button Sizes
>    In the second screenshot, the buttons seem larger than necessary. Do we really need them at that size?
>    I think we could minimize the button size so that it fits just around the content for a cleaner look.

I agree with @jagpreetrahi suggestions. 

CC: @vtushar06

vtushar06 commented:
sure @Utkarsh-123github, will move with same approach.

vtushar06 commented:
> hey @Honyii @vtushar06 , Can we show the participant number dynamically instead of statically? It would enhance the page view. What do you think about it??

yeah sure we can. @Utkarsh-123github what do you think about this??

vtushar06 commented:
> Hey @vtushar06, I hope you’re doing well! I just checked the provided outcomes from your side (via screenshots) and haven’t explored the code yet, but I have a few suggestions:
> 
> 1. Navbar Layout
>    In the first screenshot, should we add more spacing between the logo and the list items in the navbar?
>    Also, regarding the conference data (days, sessions, etc.) on the right side: currently, the spacing feels a bit too large. Would it look better if we displayed them horizontally instead of stacking with so much space?
> 2. Button Sizes
>    In the second screenshot, the buttons seem larger than necessary. Do we really need them at that size?
>    I think we could minimize the button size so that it fits just around the content for a cleaner look.

@jagpreetrahi for button size, are you talking about Sponsor?

jagpreetrahi commented:
No, I am talking  about two buttons with content as [ 9 december & main track].

vtushar06 committed:
updated button size and nav spacing

vtushar06 commented:
Hii @Utkarsh-123github, I have corrected spacing between logo and nav items and also buttons sizes, can you please look into this whenever possible and updated screenshots in description. Sorry for delays.
Thanks

Honyii commented:
@jagpreetrahi @vtushar06 pls any new additions to the site (just like displaying paqrticipant number dynamically), pls add it to the doc as suggestions, we will discuss it in our next call and I'd create another issues based on priority. The goal now is to have the previous issues deployed.

jagpreetrahi commented:
Alright I do

Honyii commented:
@vtushar06 pls can you update the date to Dec 11 and also the page title to reflect "25" as in 2025 and not "24" as in 2024.
Nice work, thanks

vtushar06 commented:
ofcourse @Honyii will update.

vtushar06 committed:
updated date and title

vtushar06 commented:
Hii @Honyii, necessary changes are made to update date to Dec 11 and title to 25 and now they are ready to get merge.
